src/licensedcode/data/licenses/ecfonts-1.0.LICENSE

```diff
@@ -0,0 +1,81 @@
+% copyrite.txt
+%
+% (c) Copyright 2012 J"org Knappen
+%
+% This file is part of ecfonts version 1.0
+
+
+                Copyright notice to the ec fonts
+
+(c) Copyright 1995, 1996, 1997, 2012 J"org Knappen, Mainz
+(c) Copyright 1990, 1992 Norbert Schwarz, Ruhr-Universit"at Bochum
+
+
+C O P Y R I G H T (2012)
+------------------------
+
+This distribution is copyright 1990--2012 Norbert Schwarz and Joerg Knappen.
+
+
+WARRANTY
```

Comment:
Can you remove all the text above this `WARRANTY` line? this is notice text that is not part of the license text proper